src/utils/auth/auth.utils.ts

```diff
@@ -270,7 +270,7 @@ export function getPopupResponse<T>(authID: string, popupWindowTabID: number) {
       } else if (isAuthErrorResult(data)) {
         log(LOG_GROUP.AUTH, `auth_result for authID = "${authID}" = Error (${data.error})`);
 
-        reject(data.error);
+        reject(new Error(data.error));
```

Comment:
Otherwise, background scripts checking for errors might not be able to handle it properly, like it was happening in `dispatch.background.ts`:

```
    if (isError(err) && err.message === ERR_MSG_USER_CANCELLED_AUTH) {
      throw err;
    }
```